showcase/app/templates/components/side-nav.hbs

```diff
@@ -876,6 +877,209 @@
 
 <Shw::Divider />
 
+<section data-test-percy>
+  <Shw::Text::H2>Child interactive components</Shw::Text::H2>
+
+  <Shw::Text::H3>Buttons within SideNav</Shw::Text::H3>
+
+  <Shw::Text::H4>States</Shw::Text::H4>
+
+  <Shw::Text::Body>Secondary</Shw::Text::Body>
+  <Shw::Flex as |SF|>
+    {{#let (array "default" "hover" "active" "focus" "disabled") as |states|}}
+      {{#each states as |state|}}
+        <SF.Item @label={{state}}>
+          <div class="hds-side-nav shw-component-set-max-content-width">
```

Comment:
that said, if you want to test this option, do a quick test and see if the outcome is actually simpler/better that using CSS, it may be that it's actually more complex

Reply:
I refactored & pushed up. I think it's ok. (The css selector becomes more complex but I think it's still pretty readable so hopefully not too hard to understand what's going on)